src/Sylius/Bundle/VariationBundle/Form/Type/OptionValueTranslationType.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\VariationBundle\Form\Type;
+
+use Sylius\Bundle\ResourceBundle\Form\Type\AbstractResourceType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * @author Vincenzo Provenza <vincenzo.provenza89@gmail.com>
+ */
+class OptionValueTranslationType extends AbstractResourceType
+{
+    /**
+     * @var string
+     */
+    protected $variableName;
```

Comment:
I think that according to convention in all *Sylius*, it should be ``$subject``.